### PR TITLE
Adding Tutorial: Migration from CVS to Git

### PR DESCRIPTION
**Pull request by hlbp**
I have read and understood the Contributor's Certificate of Origin available at the end of
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: Helmer Barcos helmer@barcos.co

### CODE REVIEW

File: tutorials/how-to-migrate-a-cvs-repository-to-git/01.en.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
+---
+SPDX-License-Identifier: MIT
+path: "/tutorials/how-to-migrate-a-cvs-repository-to-git"
+slug: "how-to-migrate-a-cvs-repository-to-git"
+date: "2021-08-02"
+title: "How to migrate a CVS repository to Git"
+short_description: "In this tutorial I will show you how to login to your Hetzner instance with a SSH Key"
```
Comment (rook59) [inline tutorials/how-to-migrate-a-cvs-repository-to-git/01.en.md:7]: Please update the description to match this tutorial

File: tutorials/how-to-migrate-a-cvs-repository-to-git/01.en.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+- Git supports SSH, HTTP/HTTPS.
+
+**Prerequisites**
+
+- Root access to the CVS Server
+- [GNU Make](https://de.wikipedia.org/wiki/GNU_Make) installed.
+- [Git](https://de.wikipedia.org/wiki/Git) installed.
+- A couple of waiting hours depending on how old the history of the CVS is.
+
+## Step 1 - Clone the [cvs2svn]() project and install it.
```
Comment (rook59) [inline tutorials/how-to-migrate-a-cvs-repository-to-git/01.en.md:41]: > [cvs2svn]()

What's this? Did you want to link anything?

File: tutorials/how-to-migrate-a-cvs-repository-to-git/01.en.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+- Access as root on the host location where the cvs repo is located.
+- Clone and build the cvs2svn project. This commnd will install the [cvs2svn](https://www.mcs.anl.gov/~jacob/cvs2svn/cvs2svn.html) and the [cvs2git](https://www.mcs.anl.gov/~jacob/cvs2svn/cvs2git.html) tools.
+
+  ```bash
+  git clone https://github.com/mhagger/cvs2svn.git
+  cd cvs2svn && make build && cd ...
```
Comment (rook59) [inline tutorials/how-to-migrate-a-cvs-repository-to-git/01.en.md:48]: build is not available

```
Supported make targets:
    man -- Create manpages for the main programs
    install -- Install software using distutils
    dist -- Create an installation package
    check -- Run cvs2svn tests
    pycheck -- Use pychecker to check cvs2svn Python code
    clean -- Clean up source tree and temporary directory

**Review by rook59 — CHANGES_REQUESTED** [review]
Thank you for your submission.  
Why do you use cvs2svn instead of https://git-scm.com/docs/git-cvsimport?

**Comment by hlbp** [comment]
@rook59  thanks for your feedback. I updated the MR in order to attend your comments. 
 There is a warning on the website from git for the `git-cvsimport`. That is the reason why I used `cvs2git`. I also expended a couple of hours trying with `git-cvsimport` without any success.

> WARNING: git cvsimport uses cvsps version 2, which is considered deprecated; it does not work with cvsps version 3 and later. If you are performing a one-shot import of a CVS repository consider using cvs2git or cvs-fast-export.

File: tutorials/how-to-migrate-a-cvs-repository-to-git/01.en.md (hunk 1 above)
Comment (hlbp) [reply tutorials/how-to-migrate-a-cvs-repository-to-git/01.en.md]: done

File: tutorials/how-to-migrate-a-cvs-repository-to-git/01.en.md (hunk 2 above)
Comment (hlbp) [reply tutorials/how-to-migrate-a-cvs-repository-to-git/01.en.md]: Yes! the author's git repo of the project.

File: tutorials/how-to-migrate-a-cvs-repository-to-git/01.en.md (hunk 3 above)
Comment (hlbp) [reply tutorials/how-to-migrate-a-cvs-repository-to-git/01.en.md]: updated

**Review by rook59 — APPROVED** [review]
LGTM

**Pull request merged**